app/konnect/org-management/audit-logging/reference.md

```diff
@@ -5,10 +5,38 @@ content_type: reference
 beta: true
 ---
 
-{{site.konnect_short_name}} logs events in [ArcSight CEF Format](https://docs.centrify.com/Content/IntegrationContent/SIEM/arcsight-cef/arcsight-cef-format.htm).
+## Log overview
+{{site.konnect_short_name}} captures three types of events:
+
+* Authentication: triggered when a user attempts to log into the Konnect web application or use the Konnect API via a personal access token. Also triggered when a system account access token is used.
+* Authorization: triggered when a permission check is made for a user or system account against a resource.
+* Access logs: triggered when a request is made to the Konnect API.
+
+## Log format
+
+{{site.konnect_short_name}} delivers log events in [ArcSight CEF Format](https://docs.centrify.com/Content/IntegrationContent/SIEM/arcsight-cef/arcsight-cef-format.htm). Webhook calls include a batch of events; each event is formatted in CEF and separated by a newline. The `Content-Type` is `application/text`.
+
+To minimize payload size, the message body is compressed.  The `Content-Encoding` is `application/gzip`.
+
+All log entries include the following attributes:
+
+Property | Description
+---------|-------------
+Timestamp | Time and date of the event in UTC.
+`rt` | Milliseconds since Unix epoch.
+`src` | The IP address of the request originator.
+`org_id` | The originating organization ID.
+`principal_id` | The user ID of the user that performed the action.
+`actor_id` | If another user performed the request on behalf of the principal, the acting user's ID is provided here. (See note.)
+`trace_id` | The correlation ID of the request. Use this value to find all log entries for a given request.
+`user_agent` | The user agent of the request: application, operating system, vendor, and version.
+
+_Note: Only an authorized Kong employee may act on behalf of another user._ 
```

Comment:
question: Docs team, we include the `actor_id` so the customer will know if we were acting on their behalf. They won't be able to correlate the ID in this field to any of their records. How do we describe this in a way that is more reassuring than alarming?

Reply:
I am following up on whether we should GA audit logs with `actor_id`. Will report back.